### PR TITLE
Tezos Wrappers Updated

### PR DESCRIPTION
**Pull request by namesty**

### CODE REVIEW

File: protocol/tezos/harbinger-wrapper/src/__tests__/e2e/get-asset-data.spec.ts [hunk 1]
```diff
@@ -1,58 +1,47 @@
 import path from "path"
-import { tezosPlugin } from "@blockwatch-cc/tezos-plugin-js"
-import { Web3ApiClient } from "@web3api/client-js"
-import { buildAndDeployApi, initTestEnvironment, stopTestEnvironment } from "@web3api/test-env-js"
+import { PolywrapClient } from "@polywrap/client-js"
+import { buildWrapper } from "@polywrap/test-env-js"
+import { AssetCandle, Providers } from "../../wrap"
```
Comment (Niraj-Kamdar) [inline protocol/tezos/harbinger-wrapper/src/__tests__/e2e/get-asset-data.spec.ts:4]: Use app codegen instead

File: protocol/tezos/harbinger-wrapper/src/__tests__/e2e/get-asset-data.spec.ts [hunk 2]
```diff
@@ -81,40 +70,9 @@ describe("Query", () => {
...
-      expect(response.data?.getAssetData.close).toBeDefined()
-      expect(response.data?.getAssetData.volume).toBeDefined()
-      expect(response.data?.getAssetData.endPeriod).toBeDefined()
-      expect(response.data?.getAssetData.startPeriod).toBeDefined()
-    })
-
     it("should get asset data for `XTZ-USD` on custom network", async () => {
-      const response =  await client.query<{ getAssetData: QuerySchema.AssetCandle}>({
-        uri: ensUri,
+      const response =  await client.query<{ getAssetData: AssetCandle}>({
```
Comment (Niraj-Kamdar) [inline protocol/tezos/harbinger-wrapper/src/__tests__/e2e/get-asset-data.spec.ts:74]: don't use `client.query` anymore. I'd suggest to use app codegen whenever possible and if not then invoke.

File: protocol/tezos/harbinger-wrapper/.gitignore [hunk 3]
```diff
@@ -6,5 +6,6 @@ coverage
 *.log
 w3
 .w3
+.polywrap
```
Comment (Niraj-Kamdar) [inline protocol/tezos/harbinger-wrapper/.gitignore:9]: add wrap in gitignore

File: protocol/tezos/harbinger-wrapper/package.json [hunk 4]
```diff
@@ -1,32 +1,28 @@
...
+  "version": "0.1.1",
   "license": "MIT",
   "scripts": {
     "reset": "yarn clean && yarn build",
-    "clean": "npx rimraf ./node_modules ./build ./coverage ./.w3 ./**/w3",
-    "build": "yarn build:web3api",
-    "build:web3api": "npx w3 build",
+    "clean": "npx rimraf ./node_modules ./build ./coverage ./.wrap ./**/wrap",
+    "build": "yarn build:polywrap",
+    "build:polywrap": "npx polywrap build",
```
Comment (Niraj-Kamdar) [inline protocol/tezos/harbinger-wrapper/package.json:11]: Why have separate build:polywrap ?

File: README.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ Various integrations that have been developed using Polywrap. Integrations are d
 | [Near JS Plugin (limited)](./protocol/near/plugin-js) | Complete | Not deployed. See the [full Near wrapper specification](near/Near%20Polywrapper%20Specification.md) |
 | [Tezos Core Plugin](./protocol/tezos/plugin-js)     		   | Complete | NPM: [tezos-plugin-js](https://www.npmjs.com/package/@blockwatch-cc/tezos-plugin-js)    |
 | [Tezos Domains JS Plugin](./protocol/tezos/tezos-domains-plugin-js)     | Complete | NPM: [tezos-domains-plugin-js](https://www.npmjs.com/package/@blockwatch-cc/tezos-domains-plugin-js)    |
-| [Tezos Domains Wrapper](./protocol/tezos/tezos-domains-wrapper)     | Complete | w3://ipfs/QmeeHVsKoo7pkYNeTFtFYuRZAHoijjei729NBNo2YCEEXj    |
+| [Tezos Domains Wrapper](./protocol/tezos/tezos-domains-wrapper)     | Complete | w3://ipfs/QmfXgYfxG7Uw7RCGLsAYFCuPcr7xqRePhTgMJVCdGiRMcZ    |
```
Comment (Niraj-Kamdar) [inline README.md:25]: rename w3 -> wrap

**Review by Niraj-Kamdar — CHANGES_REQUESTED** [review]
update deploy manifests to origin

**Comment by namesty** [comment]
> update deploy manifests to origin

Yep, done

**Review by Niraj-Kamdar — APPROVED** [review]

**Pull request merged**